terraform/api/aws/s3.tf

```diff
@@ -1,13 +1,19 @@
 resource "aws_s3_bucket" "storage" {
-  acl           = "private"
```

Comment:
> S3 Bucket canned ACL can be configured in either the standalone resource [aws_s3_bucket_acl](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_acl) or with the deprecated parameter acl in the resource aws_s3_bucket. Configuring with both will cause inconsistencies and may overwrite configuration.

The `acl` param was deprecated so I changed to use the `aws_s3_bucket_acl` resource.